eng/scripts/Prepare-Docker.ps1

```diff
@@ -55,7 +55,7 @@ Remove-Item -Path $OutputPath -Recurse -Force -ErrorAction SilentlyContinue -Pro
 Push-Location $RepoRoot
 try {
     $buildInfo = Get-Content $BuildInfoPath -Raw | ConvertFrom-Json
-    $platformName = "linux-x64"
+    $platformName = "linux-x64-untrimmed"
```

Comment:
@hallipr Perhaps I'm missing something, but doesn't this mean that all Docker builds use untrimmed? I thought we were adding an additional one not replacing the existing one.